modal/runner.py

```diff
@@ -373,6 +373,7 @@ async def _deploy_app(
     show_progress=True,
     environment_name: Optional[str] = None,
     public: bool = False,
+    tag: Optional[str] = None,
```

Comment:
FWIW `tag` could also just be typed as `str` with an empty default since it's going to get sent via a protobuf anyway, that will (very slightly) simplify the rest of the code that uses it.

Reply:
Sounds good will do